tslint.json

```diff
@@ -139,6 +139,6 @@
     "use-pipe-transform-interface": true,
     "component-class-suffix": true,
     "directive-class-suffix": true,
-    "invoke-injectable": true
+    "invoke-injectable": true,
```

Comment:
why adding this comma ? this is not standard JSON (your editor may sometimes require "trailing-comma" but valid JSON is without.

Reply:
👍